react/features/base/jitsi-local-storage/setup.web.js

```diff
@@ -41,7 +55,9 @@ function shouldUseHostPageLocalStorage(urlParams) {
         return true;
     }
 
-    if (browser.isWebKitBased()) { // Webkit browsers don't persist local storage for third-party iframes.
+    if (browser.isWebKitBased() && _inIframe()) {
```

Comment:
maybe we can add the check for everything? Like if we are not in iframe to always return false early. WDYT?

Reply:
why?

Reply:
ah wait, then we'd have 2 checks... I like the combined one better I think :-)

Reply:
Yes, but that's the default (the last statement in the function). I think it's easier to read if all the conditions are for when to return true. Coincidentally that'ss how it was written.